### ISSUE TITLE
Some manifests are not diffed

### ISSUE THREAD

**Issue by SeMeKh** [databus23/helm-diff#162]
Test Scenario
---
Clone the repository at: https://github.com/SeMeKh/helm-diff-reproduce-diff-issue

1. `helm3 install myrelease mychart`
2. `helm3 diff upgrade myrelease mychart --set "service.port=81"`
The diff command shows no changes, although there should be.
3. `helm3 upgrade myrelease mychart --set "service.port=81"`
The service port changes indeed.

**Comment by databus23** [comment]
Could you please provide the output of the following command:
`helm3 version`
`helm3 plugin list`

Thanks.

**Comment by SeMeKh** [comment]
```
$  helm3 version
version.BuildInfo{Version:"v3.0.0-rc.1", GitCommit:"ee77ae3d40fd599445ebd99b8fc04e2c86ca366c", GitTreeState:"clean", GoVersion:"go1.13.3"}
$ helm3 plugin list
NAME    VERSION         DESCRIPTION                           
diff    3.0.0-rc.6      Preview helm upgrade changes as a diff
```

**Comment by databus23** [comment]
Ok that's reasonable and should work.
Could you provide the output of `helm3 diff upgrade myrelease mychart --set "service.port=81 --debug`?

**Comment by SeMeKh** [comment]
```
Executing helm3 get manifest myrelease --namespace default
Executing helm3 template myrelease mychart --namespace default --set service.port=81
Executing helm3 get hooks myrelease --namespace default
```
And result of executing the 2nd command is:
```
---
# Source: mychart/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: myrelease-mychart
spec:
  type: ClusterIP
  ports:
    - port: 81
      targetPort: http
      protocol: TCP
      name: http
```

**Comment by databus23** [comment]
Ok but can you run the diff command with `—debug`

**Comment by databus23** [comment]
Sorry accidentally closed this issue

**Comment by SeMeKh** [comment]
The first block is the output of running `helm3 diff upgrade myrelease mychart --set "service.port=81" --debug`
Am I missing something?

**Comment by databus23** [comment]
I‘m confused. The output should be very verbose with helm and plugin binary spewing out a lot of debug info. Specifically the diff plugin should print out the helm commands it executed under the hood.

**Comment by databus23** [comment]
Ah ok sorry my bad I understand now

**Comment by SeMeKh** [comment]
I might add that I can diff lots of other charts successfully. I'm starting to suspect that only the first manifests' diff is ignored e.g. if I have a chart containing multiple manifests, and I apply a change, only the first manifest does not show up as changed.

**Comment by databus23** [comment]
Oh, that is interesting.
I'm starting to suspect that something changed with helm3 that make this not true anymore:
https://github.com/databus23/helm-diff/blob/master/manifest/parse.go#L91

**Comment by databus23** [comment]
I think I got it. The problem is the leading newline in this seperator: https://github.com/databus23/helm-diff/blob/master/manifest/parse.go#L18

Let me chew on it...

**Comment by databus23** [comment]
I believe the reported issue is closed by https://github.com/databus23/helm-diff/commit/c583d071860a7619d8b9dabed101348c80d1742f.

I also just released 3.0.0-rc.7.

Thanks for reporting this issue.
